freyja/_cli.py

```diff
@@ -242,8 +246,46 @@ def dash(agg_results, metadata, title, intro, thresh, headercolor,
         titleText = ''.join(f.readlines())
     with open(intro, "r") as f:
         introText = ''.join(f.readlines())
+    if config is not None:
+        with open(config, "r") as f:
+            try:
+                config = yaml.safe_load(f)
+            except yaml.YAMLError as exc:
+                raise ValueError('Error in config file: ' + str(exc))
+
+    # Download lineages.yml from cov-lineage repository.
+    # and save it as lineages.yml
+    r = requests.get('https://raw.githubusercontent.com/cov-lineages/' +
+                     'lineages-website/master/data/lineages.yml')
+    if r.status_code == 200:
+        print('Downloading lineages.yml')
+        with open('freyja/data/lineages.yml', 'w+') as f:
```

Comment:
the path can't be hard coded here. should instead look like ```os.path.join(locDir, 'data/lineages.yml')```

Reply:
Done